stdlib/public/SwiftShims/swift/shims/CoreFoundationShims.h

```diff
@@ -71,6 +71,10 @@ _swift_stdlib_NSStringGetCStringTrampoline(id _Nonnull obj,
                                            _swift_shims_CFIndex maxLength,
                                            unsigned long encoding);
 
+SWIFT_RUNTIME_STDLIB_API
+const __swift_uint8_t * _Nullable
+_swift_stdlib_NSStringUTF8StringTrampoline(id _Nonnull obj);
```

Comment:
My kingdom for a way to force swiftc to generate an objc_msgSend_super call so I don't have to do this ridiculous hoop jumping